frontend/views/containers/proposals/ProposalsWidget.vue

```diff
@@ -164,12 +178,18 @@ export default ({
         'remove-member': { toRemove: true }
       }
 
-      if (itemId === 'add-new-member' && this.groupShouldPropose) {
+      const isWelcomeInviteExpired = this.currentWelcomeInvite.expires < Date.now()
+      if (itemId === 'add-new-member' && (this.groupShouldPropose || isWelcomeInviteExpired)) {
```

Comment:
Note that your other PR adds code related to `groupShouldPropose`: https://github.com/okTurtles/group-income/pull/1643/files#diff-e12f80a0ba7af9f7795e718cfba169d8a5dee2e91542d13e203c242fff48dc88R104

Is `groupShouldPropose` ever checked without also checking the expiry of the invite? Maybe `groupShouldPropose` should be updated itself so that it accounts for the expiry of the invite so that everything works well with the changes in PR #1643 too?

Reply:
`groupShouldPropose` only describes if there are 3 members in the group, and if users should propose to do something. And it could be used for all the proposals, so I don't think it should be updated.

Even though the user is not a group creator, he can add members if the initial invite link is expired (30 days by default) and there are less than 3 members in the group.